piker/brokers/binance.py

```diff
@@ -311,14 +300,15 @@ async def stream_messages(ws):
     timeouts = 0
     while True:
 
-        with trio.move_on_after(5) as cs:
+        with trio.move_on_after(3) as cs:
```

Comment:
@guilledk just want to make sure there was no reason this had to be a longer wait 🤔